### DISCUSSION TITLE
Themes

### DISCUSSION THREAD

**Discussion by biscuitdelicious** [ricklamers/gpt-code-ui#15] [Ideas]
Hi,

Wondering if adding the option to choose from **dark/white theme** would be a cool feature since the only way to do it is via an extension.

I gave an idea on where to place it so that it looks good. 

Thank you!

![2023-05-22 - 07-41 screenshot@2x](https://github.com/ricklamers/gpt-code-ui/assets/115494712/2403d35e-1308-480b-96f3-6a8060aa34a5)

**Comment by ricklamers** [comment]
This would be a great first issue for someone to contribute!

**Reply by biscuitdelicious** [reply]
May I work on it? 

I'll take a look first at the [guide for contributions](https://github.com/ricklamers/gpt-code-ui/blob/main/.github/CONTRIBUTING.md) then work on it if no one is interested in.

**Reply by ricklamers** [reply]
Whoever lands the first PR that’s approved :) Feel free to give it a go 💯
